docs/api/ref/GetAgent.md

```diff
@@ -0,0 +1,224 @@
+---
+id: GetAgent
+sidebar_label: GetAgent
+title: GetAgent
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+The `getAgent()` function is the recommended way for **web applications** to connect to an FDC3 Desktop Agent. The function is exported by the [`@finos/fdc3`](https://www.npmjs.com/package/@finos/fdc3) NPM module and returns a [Desktop Agent](./DesktopAgent) implementation:
+
+<Tabs groupId="lang">
+<TabItem value="ts" label="TypeScript">
+
+```ts
+import { getAgent, DesktopAgent } from "@finos/fdc3";
+
+try {
+    const fdc3: DesktopAgent = await getAgent();
```

Comment:
I understand why you've written the example this way.  Probably because you wanted to show how you can use the function return value to obtain a reference to the DA directly, and you also wanted to keep it simple by omitting params from this basic example.

But it feels like it's mixing too different concepts (the new way and the old way) together.  Setting a constant (after the function has returned) with same name as the property of the window object (window.fdc3) that was already set by default by the function is just very very odd.

Reply:
Well... there was discussion about getting away from `window.fdc3` and this shouldn't cause conflicts. When this const is in scope it will dominate ... and ... its the same reference, plus all the code examples everywhere are `fdc3.<api call name>()`.

I guess the alternative is:

```ts
try {
    const desktopAgent: DesktopAgent = await getAgent();
    //do FDC3 things here
} catch (e: AgentError) {
    // Failed to connect
}
```

That is a much bigger break from the past - but might mean we should make more changes to examples throughout the FDC3 docs?

I've changed it in 3c087a84cfb49641e45c8f56a036d2a118856849 in both the GetAgent and supported platforms pages - could you take a look and let me know if you think we should change other examples in the API overview and reference?